{"uuid":"fb42ea27-fa27-46f6-ba0b-bc2dcefaa697","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["4515d46a-dba3-48cd-ace1-b1c97c6132bb"],"befores":[],"afters":[{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687577055430,"stop":1687577055431},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687577055624,"stop":1687577055624},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687577055708,"stop":1687577055708},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687577055744,"stop":1687577055744},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687577055751,"stop":1687577055751}],"start":1687577055329,"stop":1687577055751}